io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
```

Comment:
are you sure you need the GNOME platform?

Reply:
yes, vimix uses gtk

Reply:
it is in the freedesktop SDK.

Reply:
Well, you tell me !

Maybe thus indeed, if vimix flatpak rebuilds GStreamer library (otherwise provided in Gnome platform as indicated 
[here](https://docs.flatpak.org/en/latest/available-runtimes.html?highlight=org.gnome.Platform#gnome) ), the Freedesktop runtime is more adapted?

Reply:
Back after having tried to run flatpak with `org.freedesktop.Platform` and `org.freedesktop.Sdk` : GStreamer fails to build.

Running Flatpak with `org.gnome.Platform` and `org.gnome.Sdk` does indeed builds with GStreamer, but the SRT plugins are not available (srtsink and srtsrc).

So, your help is welcome here: I just want that vimix builds with SRT lib support in GStreamer. 

Suggestion: build only bad plugins of GStreamer ?

Reply:
With the following, only gst-bad and gst-ugly plugins are recompiled and vimix has everything it needs

```
        {
            "name": "gstreamer",
            "buildsystem": "meson",
            "builddir": true,
            "config-opts": [
               "-Ddoc=disabled",
               "-Dpython=disabled",
               "-Dges=disabled",
               "-Drs=disabled",
               "-Dgst-examples=disabled",
               "-Drtsp_server=disabled",
               "-Ddevtools=disabled",
               "-Dlibav=disabled",
               "-Dbase=disabled",
               "-Dgood=disabled",
               "-Dgpl=enabled",
               "-Dbad=enabled",
               "-Dgst-plugins-bad:srt=enabled",
               "-Dugly=enabled",
               "-Dgst-plugins-ugly:x264=enabled"
            ],
            "sources": [
              {
                "type": "git",
                "tag": "1.20.4",
                "commit": "642730bd67d3c23ac68bd328fbddd191a7a43f25",
                "url": "https://gitlab.freedesktop.org/gstreamer/gstreamer.git",
                "disable-submodules": false
              }
            ]
        },
```

NB: this way, the gstreamer plugins and libraries provided in org.gnome.Platform are used, and only extra ones are added.